sos/plugins/openvswitch.py

```diff
@@ -147,24 +236,81 @@ def setup(self):
                             "ovs-ofctl -O %s dump-ports-desc %s" % (flow, br)
                         ])
 
-        # Gather info on the DPDK mempools associated with each DPDK port
-        br_list_result = self.collect_cmd_output("ovs-vsctl -t 5 list-br")
-        if br_list_result['status'] == 0:
-            for br in br_list_result['output'].splitlines():
-                port_list_result = self.exec_cmd(
+                port_list_result = self.collect_cmd_output(
```

Comment:
You need extra collecting the `"ovs-vsctl -t 5 list-ports %s"` command, that you replace `exec_cmd` (as in upstream) by `collect_cmd_output`?